src/crypto/tls/handshake_client_tls13.go

```diff
@@ -343,6 +346,8 @@ func (hs *clientHandshakeStateTLS13) processHelloRetryRequest() error {
 func (hs *clientHandshakeStateTLS13) processServerHello() error {
 	c := hs.c
 
+	hs.handshakeTimings.ReadServerHello = hs.handshakeTimings.elapsedTime()
```

Comment:
Should this go after the (HRR and the) SH has been processed?

Reply:
Hmm... presumably after SH.